client/blocks/reader-post-card/tag-link.jsx

```diff
@@ -31,7 +31,8 @@ class TagLink extends Component {
 
 	render() {
 		const { tag } = this.props;
-		const path = addLocaleToPathLocaleInFront( `/tag/${ encodeURIComponent( tag.slug ) }` );
+		const decodedSlug = decodeURIComponent( tag.slug ); // Other languages may contain non-ASCII characters so we need to decode it.
+		const path = addLocaleToPathLocaleInFront( `/tag/${ decodedSlug }` );
```

Comment:
I get that we want to display the `decodedSlug` in the button content so it is human readable. But Im not sure I understand why we are bypassing encodeURIComponent for the href, as using the decoded version could cause various issues depending on the decoded slug.

Reply:
I have noticed that the browser was encoding the url therefore thought maybe its safe to ignore it. But now after your comment I think its safe to have it (reverted in [6d43c8a](https://github.com/Automattic/wp-calypso/pull/102677/commits/6d43c8adf0a0cd90d649761582daff8244a7b546))

Reply:
> I have noticed that the browser was encoding the url therefore thought maybe its safe to ignore it. 

Yeah the browsers are weird. In some circumstances they use the encoded but render the decoded version in the address bar for readability.

Reply:
> (reverted in [6d43c8a](https://github.com/Automattic/wp-calypso/pull/102677/commits/6d43c8adf0a0cd90d649761582daff8244a7b546))

Were still missing the encode wrapper that was there before.

Reply:
Oops, should be fixed now.